uniffi_bindgen/src/bindings/kotlin/templates/ObjectTemplate.kt

```diff
@@ -3,7 +3,7 @@
 {{- self.add_import("java.util.concurrent.atomic.AtomicLong") }}
 {{- self.add_import("java.util.concurrent.atomic.AtomicBoolean") }}
 
-public interface {{ type_name }}Interface {
+public interface {{ type_name }}Interface : Disposable {
```

Comment:
Can you please explain a little more about why `Disposable` is now needed on interfaces. I think we are probably going to move *away* from `Disposable`, albeit with an opt-in for `AutoCloseable`